week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,18 +1,32 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// Add comments to explain what this function does. You're meant to use Google!,  
 function getNumber() {
     return Math.random() * 10;
 }
+// A Math.random(), only displays "floting points"(witch is baseicly just numbers using to show result), 
+// numbers that are between 0 -1, 0 is shown but not 1. With approximantly unchangeble distribution.
+// You can then change those numbers to whatever you want. Then once you changed the scale.
+// It chouses a number between the scale at random, again not incloding the maxiem number but the minimem (in this case 0)
+// witch can also be included. However, in this case there is a " * 10" on line 3.
+// This means that what ever the algorithom choses the number, it will get multipiled by 10.
```

Comment:
Can you add a sentence saying what exactly `getNumber` will return? What's the lowest and highest value it could return?